src/app/downloads/downloads.component.ts

```diff
@@ -187,7 +188,8 @@ export class DownloadsComponent implements OnInit, OnDestroy {
   isEpisodesChanged(state: FilterModel): boolean {
     return state.episodes &&
       (!this.filter.episodes ||
-      !state.episodes.map(e => e.id).every(id => this.filter.episodes.map(e => e.id).indexOf(id) !== -1));
+      !state.episodes.map(e => e.id).every(id => this.filter.episodes.map(e => e.id).indexOf(id) !== -1) ||
+      !this.filter.episodes.map(e => e.id).every(id => state.episodes.map(e => e.id).indexOf(id) !== -1));
```

Comment:
#41 so if an episode is unselected and reselected again, the metrics will be re-fetched and included